src/System/executable_query.rs

```diff
@@ -70,6 +75,27 @@ impl ExecutableQuery for RefinementExecutor {
     }
 }
 
+/// Used to store input for the reachability checker
+pub struct ReachabilityExecutor {
+    pub sys: TransitionSystemPtr,
+    pub s_state: State,
+    pub e_state: State,
```

Comment:
Documentation comments on these fields would be nice

Reply:
Fixed